### ISSUE TITLE
disable=too-few-public-methods doesn't work for empty class

### ISSUE THREAD

**Issue by roelvandergoot** [PyCQA/pylint#2913]
### Steps to reproduce
1. Create file base.py with the following content:

<pre>
"""Module doc string."""

class Base:
    """My base class."""
    # pylint: disable=too-few-public-methods
</pre>

2. Run pylint base.py

### Current behavior
<pre>
************* Module base
base.py:3:0: R0903: Too few public methods (0/2) (too-few-public-methods)

---------------------------------------------------------------------
Your code has been rated at 0.00/10
</pre>

### Expected behavior

<pre>

---------------------------------------------------------------------
Your code has been rated at 10.00/10
</pre>

### pylint --version output

<pre>
pylint 2.3.1
astroid 2.2.5
Python 3.7.3 (default, Apr  3 2019, 05:39:12)
[GCC 8.3.0]
</pre>

**Comment by PCManticore** [comment]
Thanks for the report.

**Comment by PCManticore** [comment]
Alright, the following should work:
```
class Base: # pylint: disable=too-few-public-methods
    """My base class."""
```

The given example doesn't work because the message pragma need to incorporate the object for which you are disabling / enabling a message. When used inside the body of the class, the pragma  control does not get applied as the message is emitted on the class itself (thus the line of `class Base` definition) rather than the body of the class (which would be a different line altogether).

**Comment by roelvandergoot** [comment]
Claudio,

Why can I put said pragma inside a non-empty class then?

Cheers :),
Roel.

**Comment by Pierre-Sassoulas** [comment]
The error happens on the class, in  the code of the class it's too late to disable it.

There is two way to do this:
```python
# pylint: disable=too-few-public-methods
class Base:
    """My base class."""
# pylint: enable=too-few-public-methods


class BaseCorrect:  # pylint: disable=too-few-public-methods
    """My base class."""
```
